### PR TITLE
apc epdu

### PR DESCRIPTION
**Pull request by zerrac**
- Update apc PowerNet Mib from https://www.se.com/fr/fr/product/SFPMIB436/powernet-mib-v4.3.6/
- Add definitions to get votage, current and power from APC easy PDUs (https://www.apc.com/shop/aw/en/products/Easy-PDU-Switched-ZeroU-16A-230V-20-C13-4-C19-IEC309/P-EPDU1116S).

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

### CODE REVIEW

File: includes/definitions/discovery/apc-epdu.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+mib: PowerNet-MIB
+modules:
+    os:
+        hardware:
```
Comment (murrant) [inline includes/definitions/discovery/apc-epdu.yaml:4]: This doesn't need to be an array.  Don't these need the mib in the oid to work?
PowerNet-MIB::ePDUDeviceStatusModelNumber

File: includes/definitions/apc-epdu.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+os: apc-epdu
```
Comment (murrant) [inline includes/definitions/apc-epdu.yaml:1]: Is it worthwhile to have a different OS?

File: includes/definitions/discovery/apc-epdu.yaml (hunk 1 above)
Comment (zerrac) [reply includes/definitions/discovery/apc-epdu.yaml]: my bad, i fixed it

File: includes/definitions/apc-epdu.yaml (hunk 2 above)
Comment (zerrac) [reply includes/definitions/apc-epdu.yaml:1]: maybe i missed something but i think apc os use completely different values in the same mib. My PDUs have no info for the oids used by apc os.

**Comment by Jellyfrog** [comment]
@zerrac missing the .json from https://docs.librenms.org/Developing/os/Test-Units

**Comment by zerrac** [comment]
@Jellyfrog  done

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**